test/ng/compileSpec.js

```diff
@@ -2762,6 +2762,23 @@ describe('$compile', function() {
     }));
 
 
+    it('should handle consecutive text elements as a single text element', inject(function($rootScope, $compile) {
+      window.addMutationObserver = function() {
```

Comment:
The test, without fiddling with `window` passes without the fix on IE 11.
IE11 parses text node differently when there is a mutation observer, this is why the test add it.